README.md

```diff
@@ -7,23 +7,30 @@ We use this in the [Cloudflare Discord](https://discord.gg/cloudflaredev), come
 
 ## How do I set this up?
 
-There are a few steps to the setup but it should hopefully be pretty straightforward:
+There are a few steps to the setup, but it should hopefully be pretty straightforward:
 
 1. Clone the repo
-2. Edit `src/config.ts` - here you can set the status URL, name of the webhook, avatar and publish channel
-3. Put your IDs in `wrangler.toml`
-4. Add Discord webhook with `wrangler secret put DISCORD_WEBHOOK` \
-  4b. (optional) If you want publishing, you'll also need to add a Discord bot token with `wrangler secret put DISCORD_TOKEN`
-5. Run `npm run publish` :)
+2. Edit `src/config.ts` to set the status URL, name of the webhook, avatar and publish channel
+3. Authenticate Wrangler with `npx wrangler login`, or set the `CLOUDFLARE_ACCOUNT_ID` and `CLOUDFLARE_API_TOKEN` environment variables (or put them in `.env`)
```

Comment:
.env _works_ but i don't think we even document it. `.dev.vars` is the recommended way
```suggestion
3. Authenticate Wrangler with `npx wrangler login`, or set the `CLOUDFLARE_ACCOUNT_ID` and `CLOUDFLARE_API_TOKEN` environment variables (or put them in `.dev.vars`)
```

Reply:
🤔 Are those env values read from `.dev.vars`? I thought `.dev.vars` was used for runtime secrets?

Reply:
https://developers.cloudflare.com/workers/wrangler/system-environment-variables/ specifically calls out that you can use a `.env` file